regression/strings/java_char_array/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
```

Comment:
You are probably right. I changed these to FUTURE and added an entry in my `Makefile` to use the `-F` option: https://github.com/diffblue/cbmc/pull/371/commits/fc0b6604e3b5c1ac817495057b620415a79368e3